ctl/monitoring/monitoring.go

```diff
@@ -46,19 +47,24 @@ func NewCmd() *cobra.Command {
 		Example: `# Enable/Disable Kmesh's accesslog:
 kmeshctl monitoring <kmesh-daemon-pod> --accesslog enable/disable
 
-# Enable/Disable Kmesh's metrics and accesslog:
+# Enable/Disable Kmesh's metrics(to-service and to-pod) and accesslog:
 kmeshctl monitoring <kmesh-daemon-pod> --all enable/disable
 
+# Enable/Disable Kmesh's metrics of connections between pods:
+kmeshctl monitoring <kmesh-daemon-pod> --workloadMetrics enable/disable
+
 # If you want to change the monitoring functionality of all kmesh daemons in the cluster
 kmeshctl monitoring --accesslog enable/disable
+kmeshctl monitoring --workloadMetrics enable/disable
 kmeshctl monitoring --all enable/disable`,
 		Args: cobra.MaximumNArgs(1),
 		Run: func(cmd *cobra.Command, args []string) {
 			ControlMonitoring(cmd, args)
 		},
 	}
 	cmd.Flags().String("accesslog", "", "Control accesslog enable or disable")
-	cmd.Flags().String("all", "", "Control accesslog and metrics enable or disable together")
+	cmd.Flags().String("all", "", "Control accesslog and metrics(to-service and to-pod) enable or disable together")
```

Comment:
is this not a bool var?

Reply:
Using string is consistent with the rest of the command line.